src/components/display.js

```diff
@@ -74,19 +78,44 @@ let Display = class Display extends React.Component {
         </div>
         <div className="py12 px12">
           <div className="mb6">
-            <h2 className="txt-bold txt-m block">GDP Growth Potential</h2>
-            <p className="txt-s">{ggp}</p>
+            <h2 className="txt-bold txt-m block" data-tip data-for = "GDPTip">GDP Growth Potential
+            {/* <Link  to = '/Research' style = {{ whiteSpace: "pre" }}>{`${'                      '}`} ? </Link>  */}
```

Comment:
Rather than committing commented out code, it's better to leave those lines out of the commit or delete them entirely. Commented out code has a way of piling up, and people are usually timid about deleting others' commented lines, so it can make things cluttered over time.